packages/site/src/content/Modal/ModalNotes.mdx

```diff
@@ -131,3 +131,67 @@ function ModalWithCustomFocus() {
   );
 }
 ```
+
+## Accessibility
+
+**Role and modality**: Modal renders with `role="dialog"` and
+`aria-modal="true"`.
+
+**Naming precedence**:
+
+- If a visible heading/title is provided, the dialog is named via
+  `aria-labelledby` that references the heading element.
+- If there is no visible heading/title, you can provide `accessibleName` to set
+  `aria-label` on the dialog.
+
+### Composable Modal (v2)
```

Comment:
we haven't been referring to the composable versions of components as v2 so far. let's stick to that convention.

Reply:
ah wait I see we are doing that in here.... I'm not sure I agree with this. I'm going to tweak the copy so we don't have to refer to the old one as legacy.

it's also not really accurate to say that original Modal is not composable and the new one is. it's more composable, for sure but even it has restrictions ie. the actions.